### PR TITLE
Enforce Python code formatting use flake8

### PR DESCRIPTION
**Pull request by cjh1**
This is ready to go now, I have enable flake8 on travis. 

@cquammen can you review [this change](https://github.com/OpenChemistry/tomviz/pull/634/commits/356abad8a5a6349f7280dc3dd4d43c9bc8128d21) which was done to reduce the complexity of one of our functions ( looks like you were the original author ), the rest of the changes should just be formatting ones.

### CODE REVIEW

**Comment by cjh1** [comment]
@cryos Still a few more warnings to clear up ...

**Comment by cryos** [comment]
Still looks great to me :+1: I will let you check in with anyone else you might want to get review from. Would be good to get merged soon.

**Comment by cjh1** [comment]
@cquammen can you review [this change](https://github.com/OpenChemistry/tomviz/pull/634/commits/131156d740d430f1e4c1d7a07952bc9658acb680) was needed to reduce the complexity of vtk_itk_map()

File: tomviz/python/tomviz/utils.py [hunk 1]
```diff
@@ -31,50 +32,31 @@ def vtk_itk_map():
...
-        try:
-            _vtk_to_itk_types['vtkFloatArray'] = itk.Image.F3
-        except:
-            pass
-
-        try:
-            _vtk_to_itk_types['vtkDoubleArray'] = itk.Image.D3
-        except:
-            pass
+
```
Comment (cquammen) [inline tomviz/python/tomviz/utils.py]: Sorry, missed your initial request for reviewing this.

I don't know if this code is less complex, but it is shorter. :-) LGTM

**Comment by cjh1** [comment]
Well according to flake8 the cyclomatic complexity is less :smile: It certainly reduces the number of try except blocks. If you feel it was better before, I can revert the change and add a # noqa instead to exclude it form flake8 testing. 

If I merge this are you OK with rebasing your two PR and fixing any flake8 issues? Otherwise, I can hold off.

**Comment by cquammen** [comment]
I think it is fine. I'm fine with rebasing my other PRs and fixing up any issues that crop up.

**Comment by cjh1** [comment]
@cquammen OK merged, if you run into any issues let me know.

**Pull request merged**